po/cs.po

```diff
@@ -2343,246 +2303,209 @@ msgid "_Give Privileges"
 msgstr "_Dát privilegia"
 
 #: pynicotine/gtkgui/widgets/dialogs.py:299
-#, fuzzy
 msgid "Close"
-msgstr "Karta Zavřít"
+msgstr "Zavřít"
 
 #: pynicotine/gtkgui/widgets/dialogs.py:478
-#, fuzzy
 msgid "_Yes"
 msgstr "_Ano"
 
 #: pynicotine/gtkgui/widgets/dialogs.py:512
 #: pynicotine/gtkgui/ui/dialogs/preferences.ui:23
-#, fuzzy
 msgid "_OK"
-msgstr "OK"
+msgstr "_OK"
 
 #: pynicotine/gtkgui/widgets/filechooser.py:39
-#, fuzzy
 msgid "Select a File"
-msgstr "Výběr souboru"
+msgstr "Vybrat soubor"
 
 #: pynicotine/gtkgui/widgets/filechooser.py:174
-#, fuzzy
 msgid "Select a Folder"
-msgstr "Výběr složky"
+msgstr "Vybrat složku"
 
 #: pynicotine/gtkgui/widgets/filechooser.py:179
-#, fuzzy
 msgid "_Select"
-msgstr "Vyberte Vše"
+msgstr "_Vybrat"
 
 #: pynicotine/gtkgui/widgets/filechooser.py:196
-#, fuzzy
 msgid "Select an Image"
-msgstr "Výběr obrázku"
+msgstr "Vybrat obrázek"
 
 #: pynicotine/gtkgui/widgets/filechooser.py:203
-#, fuzzy
 msgid "All images"
 msgstr "Všechny obrázky"
 
 #: pynicotine/gtkgui/widgets/filechooser.py:241
-#, fuzzy
 msgid "Save as…"
 msgstr "Uložit jako…"
 
 #: pynicotine/gtkgui/widgets/filechooser.py:288
 #: pynicotine/gtkgui/widgets/filechooser.py:405
-#, fuzzy
 msgid "(None)"
-msgstr "(Žádné)"
+msgstr "(nic)"
 
 #: pynicotine/gtkgui/widgets/iconnotebook.py:118
-#, fuzzy
 msgid "Close Tab"
-msgstr "Karta _Close"
+msgstr "Zavřít kartu"
 
 #: pynicotine/gtkgui/widgets/iconnotebook.py:453
-#, fuzzy
 msgid "Close All Tabs?"
 msgstr "Zavřít všechny karty?"
 
 #: pynicotine/gtkgui/widgets/iconnotebook.py:454
-#, fuzzy
 msgid "Do you really want to close all tabs?"
 msgstr "Opravdu chcete zavřít všechny karty?"
 
 #: pynicotine/gtkgui/widgets/iconnotebook.py:478
-#, fuzzy, python-format
+#, python-format
 msgid "%i Unread Tab(s)"
-msgstr "Nepřečtené karty"
+msgstr "%i nepřečtených karet"
 
 #: pynicotine/gtkgui/widgets/iconnotebook.py:481
-#, fuzzy
 msgid "All Tabs"
-msgstr "Zavřít všechny karty?"
+msgstr "Všechny karty"
 
 #: pynicotine/gtkgui/widgets/iconnotebook.py:735
 #: pynicotine/gtkgui/widgets/iconnotebook.py:740
-#, fuzzy
 msgid "Re_open Closed Tab"
-msgstr "Karta _Close"
+msgstr "Znovu otevřít zavřenou kartu"
 
 #: pynicotine/gtkgui/widgets/popupmenu.py:381
-#, fuzzy, python-format
+#, python-format
 msgid "%s File(s) Selected"
-msgstr "%s Vybrané soubory"
+msgstr "%s souborů vybráno"
 
 #: pynicotine/gtkgui/widgets/popupmenu.py:418
 #: pynicotine/gtkgui/ui/userinfo.ui:490
-#, fuzzy
 msgid "_Browse Files"
-msgstr "_Browse soubory"
+msgstr "_Procházet soubory"
 
 #: pynicotine/gtkgui/widgets/popupmenu.py:421
 #: pynicotine/gtkgui/widgets/popupmenu.py:465
-#, fuzzy
 msgid "_Add Buddy"
-msgstr "Přidat kamaráda…"
+msgstr "_Přidat kamaráda"
```

Comment:
@slrslr `_P` keyboard accelerator conflicts with `"_Procházet soubory"` ("_Browse Files").

Each item within the same context menu should logically have unique `_` accelerators, otherwise multiple key presses are required to select and then activate the desired item.